dapps/shop/backend/queues/queues.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @file Queue objects for adding processing dshop jobs.
+ *
+ * If no REDIS_URL defined, will fallback to prossing jobs as the jobs are
+ * submitted, without using a queue.
+ */
+
+const { REDIS_URL } = require('../utils/const')
+
+const Queue = REDIS_URL ? require('bull') : require('./fallbackQueue')
```

Comment:
needs to be bullmq now we changed the package?

Reply:
Fixed. I miss tests. :(